adafruit_motorkit.py

```diff
@@ -52,10 +51,18 @@
 class MotorKit:
     """Class representing an Adafruit DC & Stepper Motor FeatherWing, Shield or Pi Hat kit.
 
-    Automatically uses the I2C bus on a Feather, Metro or Raspberry Pi if no I2C bus
-    is supplied.
+    :param int address: I2C address of PCA9685 PWM controller. Default address is ``0x60``.
+    :param busio.I2C: I2C bus object to use. If not specified, use ``board.I2C()``.
 
-    Alternately, if using with multiple I2C devices, you can specify the I2C bus."""
+      .. note::
+        ``board.I2C()`` uses the default I2C bus frequency of 100 kHZ. To speed up
+        motor control, use an I2C bus frequency of 400 KHz, or if available, 1 MHz.
+        The PCA9685 controller supports both of these higher speeds.
+        This will noticeably speed up stepper motor operation when many steps are requested.
+
+    :param int steppers_microsteps: Number of microsteps per step for stepper motors. Default is 16.
+    :param pwm_frequency float: defaults to 1600 Hz
```

Comment:
Looks like `float` and `pwm_frequency` switched places!

Reply:
whoops, thanks!